File: Model/Entity.cs
namespace Lab2C.Model
{
    /// <summary>
    /// Generic class used to represent entities with an unique identification
    /// </summary>
    /// <typeparam name="TId"> type of ID </typeparam>
    public class Entity<TId>
    {
        public TId Id { get; set; }
    }
}

File: Model/src/main/java/Entity.java
/*
 *  @author albua
 *  created on 28/02/2021
 */

import java.io.Serializable;

/**
 * Generic class used to represent entities with an unique identification
 * @param <Type> type ID of entity
 */
public class Entity<Type> implements Serializable {
    protected Type ID;

    public Type getID() {
        return ID;
    }

    public void setID(Type ID) {
        this.ID = ID;
    }
}
